Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -161,7 +161,7 @@
 #endif
 
 #if ENABLE(DATA_DETECTION) && PLATFORM(IOS_FAMILY)
-#import <pal/spi/ios/DataDetectorsUISPI.h>
+SOFT_LINK_PRIVATE_FRAMEWORK_OPTIONAL(DataDetectorsUI)
```

Comment:
Yes. I initially thought that this might be tied to "locked device" state, but after speaking with the linking team I don't believe that would affect the framework directories. So something else is the issue.